bicep/nested_workload-stamp-prereqs.bicep

```
@@ -0,0 +1,56 @@
+param resourceGroupLocation string
+
+resource uid_workflow 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
```

convention changed a bit the last months.


```suggestion
resource workflowManagedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
```